refactoring-code/strangler-approach/1-create-scenarios.md

```diff
@@ -58,19 +59,29 @@ import
 
 Only add console.log statements. Do not change any other part of the code. Do not add any other code.
 
-## Step 3: Setup Standalone Environment
+## Step 3: Setup Standalone Environment // new chat window; otherwise started copying over rewrite directory
 
 Configure the app-requirements folder to be standalone environment that Playwright can be run in.
 
-Do not write any tests, or make any changes outside of this folder.
+Do not write any tests inside app-requirements.
+
+Do not setup global users.
+
+Do not make any changes outside of this folder.
+
+Only setup playwright config, and any other necessary setup files.
```

Comment:
added these because rather than simple playwright config + page is running test it was doing global-setups, user tests, login/logout tests, tests that weren't working (legacy text selector, checking that `Tournaments` text exists when homepage had 2 instances)